dataactcore/models/domainModels.py

```diff
@@ -85,7 +85,7 @@ class SF133(Base):
     ending_period_of_availabil = Column(Text)
     main_account_code = Column(Text, nullable=False)
     sub_account_code = Column(Text, nullable=False)
-    tas = Column(Text, nullable=False, default=concatTas, onupdate=concatTas)
```

Comment:
We don't actually update SF133s other than through the new update command which sets `tas_id`. Removing it allows us to avoid pulling each SF133 from the db, run `concatTas` (which would produce the same result), and inserting it again. Does that make sense?